benchmarks/Benchmarks.Trace/Program.cs

```diff
@@ -6,7 +6,7 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            BenchmarkRunner.Run<SpanBenchmark>();
+            BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
```

Comment:
Can we also add the Exporters here by using `DefaultConfig.AddExporter(...)`?  At least the Datadog exporter that can't be configured from the command line...

Reply:
That's why I left the [DatadogExporter] attribute on the benchmark

Reply:
Added it to the default config

Reply:
Great!